uPortal-webapp/src/main/webapp/media/skins/respondr/common/common.less

```diff
@@ -17,17 +17,17 @@
  * under the License.
  */
  /**
-  * NOTE: this file is located outside the less/ directory due to a limitation 
+  * NOTE: this file is located outside the less/ directory due to a limitation
   * in the LESS CSS Maven Plugin's inability to traverse relative paths.
   *
   * Also, due to a limitation in the LESS CSS Maven Plugin, whitespace before an
   * @import will cause a compile error.
   */
- 
+
  /* ==========================================================================*
     Bootstrap
     ==========================================================================*/
-@import "bootstrap/bootstrap.less";
+@import "../../../../../../../node_modules/bootstrap/less/bootstrap.less";
 
```

Comment:
The dymanic-respondr-skin portlet does allow you to select the skin upon which your skin will be based.  It also gives you several color-pickers that you can use to customize your "dynamic" skin.

When you hit Submit -- after using the color-pickers -- your color choices are used to recompile Bootstrap _from source_.  I haven't looked... but (with this update) are the Bootstrap LESS files (sources) present in Tomcat in the uPortal webapp?  Will they be available in the deployed/running portal for compiling via lessc?  And if they are, is everything wired up correctly -- paths, etc. -- for iti to work?